<ui version="4.0" >
 <class>AutoAwayOptionsClass</class>
 <widget class="OptionsPageI" name="AutoAwayOptionsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AutoAwayOptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <spacer name="verticalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="chkEnable" >
     <property name="text" >
      <string>Enable auto away</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>If I am idle for</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="spnMin" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>1440</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>minutes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>set my status to</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QComboBox" name="cmbStatus" />
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRestore" >
     <property name="text" >
      <string>Restore my old status when I return</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>OptionsPageI</class>
   <extends>QWidget</extends>
   <header>options_i.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
